shared/constants/types/provision.js

```diff
@@ -15,7 +15,10 @@ export type _State = {
   codePageOtherDeviceName: string,
   codePageOtherDeviceType: 'mobile' | 'desktop',
   codePageOtherDeviceId: string,
-  codePageTextCode: HiddenString,
```

Comment:
i split this into 2 distinct values. before it could get mixed between the incoming code and outgoing code